doc/man/8/cephadm.rst

```diff
@@ -37,7 +37,7 @@ Synopsis
 | **cephadm** **run** [-h] --name NAME --fsid FSID
 
 | **cephadm** **shell** [-h] [--fsid FSID] [--name NAME] [--config CONFIG]
-                        [--keyring KEYRING] [--mount MOUNT] [--env ENV]
+                        [--keyring KEYRING] --mount MOUNT [MOUNT ...] [--env ENV]
```

Comment:
@sebastian-philipp @guits the mount parameter should use brackets here right ?

`[--mount MOUNT [MOUNT ...]]`

Reply:
that's correct